boards/iotlab-m3/dist/openocd.cfg

```diff
@@ -1,9 +1,7 @@
-interface ftdi
-ftdi_vid_pid 0x0403 0x6010
+source [find interface/ftdi/iotlab-usb.cfg]
 
-ftdi_layout_init 0x0c08 0x0c2b
-ftdi_layout_signal nTRST -data 0x0800
-ftdi_layout_signal nSRST -data 0x0400
+# use combined on interfaces or targets that can't set TRST/SRST separately
+reset_config trst_and_srst
```

Comment:
This doesn't work as is for me but adding `connect_assert_srst` works.

Reply:
Ah, adding `connect_assert_srst` also breaks the `debug` target (but this could be solved the same way as in #8856)... and now it works without it (`connect_assert_srst`).

Reply:
With `reset_config trst_and_srst connect_assert_srst` I indeed need to do a `-c 'reset halt'` before debugging. With it it works.